tracer/src/Datadog.Trace/Iast/Aspects/System.Text/StringBuilderAspects.cs

```diff
@@ -745,7 +745,7 @@ public static void SetLength(StringBuilder? target, int length)
         StringBuilderModuleImpl.FullTaintIfAnyTainted(target);
     }
 
-#if NETCOREAPP3_1_OR_GREATER
```

Comment:
There were a recent issue on that, were it was failing on netcore3.0.
Does it still a good idea to change it to `NETCOREAPP` or do we want to change it later when we fixed it?

Reply:
We definitely need to look into it and work out what the _right_ option is here - it's very possible I've got some things confused, will catch up with you as to what the _correct_ option is here! 😄